README.md

```diff
@@ -206,3 +206,19 @@ ninja layers
 ```
 
 After above operations, demo Aos layers could be found in `output/layers` folder.
+
```

Comment:
1. First line of commit message should be <= 72 symbols
2. Remove extra lines

Reply:
updated